authors/apps/articles/serializers.py

```diff
@@ -31,15 +33,16 @@ class Meta:
             "favorited_by",
             "favoritesCount",
             "favorited",
-            "average_ratings"
+            "average_ratings",
+            "read_time",
```

Comment:
Shouldn't this also be a `read_only` field?

Reply:
It is a read-only field specified by use of the SerializerMethodField. Please read more on this here: https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield